src/Backend/ArrayCache.php

```diff
@@ -32,7 +32,7 @@ public function doSave($id, $data, $lifeTime = 0)
     public function doDelete($id)
     {
         if (!$this->doContains($id)) {
-            return false;
+            return true;
```

Comment:
I haven't actually tested the other backends but I did look at the code when changing the implementation and concluded from the code review it was safe to change. The change (https://github.com/piwik/component-cache/issues/10) should make backends consistent behavior. Hopefully I didnt miss something!